### ISSUE TITLE
(Feature request) Consider right-aligning "+" button in the tree view 

### ISSUE THREAD

**Issue by jrosiek** [zadam/trilium#2421]
### Preflight Checklist

- [X] I have searched the [issue tracker](https://www.github.com/zadam/trilium/issues) for a feature request that matches the one I want to file, without success.

### Describe feature

Currently, to add a subnode in the tree view the user is required to align the mouse not only vertically, but also horizontally (because location of the "+" button depends on the length of note title). It is more more demanding from the UX standpoint. Right aligning the buttons will position them in the same place horizontally.

**Comment by wunter8** [comment]
You can add this to your personal Trilium instance with custom CSS!

1. Create a new note
2. Set type to CSS
3. Paste this code into the note:
```
.add-note-button {
    position: absolute;
    right: 10px;
}
```
4. Add attribute `#appCss` to the note
5. Reload the frontend (press `F5` or `Ctrl+R`)

**Comment by mechanarchy** [comment]
This is fixed in #3928 and should be in the next release I believe

**Comment by zadam** [comment]
Yep, closing.
